nens_auth_client/models.py

```diff
@@ -192,3 +211,26 @@ def send_email(self, request, context=None, send_email_options=None):
 
         self.email_sent_at = timezone.now()
         self.save()
+
+
+def clean_invitations(days=90):
```

Comment:
Deze default staat ook in het mgmt command. Defaults heb ik het liefste maar op één plek zodat het duidelijk is wat de echte default is. (Ik heb wel eens False en False gezien als defaults en toen was er voor hetzelfde ding nog een derde en dat was True..... )